Fixing Issue with CEFX Conept

PhilMFischer opened the pull request:
Trying to fix failing test cases on CEFX concept due to beta tag on DVLM v6 v7 problems.

Commit:
Updated the Target Platform - (Task #61)

- Updated TargetPlatform for Development

---
Task #61: Regenerate Concepts due to change in DVLM

Commit:
Removed beta from version with DVLM v6 - (Task #61)

---
Task #61: Regenerate Concepts due to change in DVLM

Commit:
Migrated CEFX concept to DVLM v7 - (Task #61)

- added is beta feature
- migrated to DVLM v7

---
Task #61: Regenerate Concepts due to change in DVLM

Commit:
Removed beta from version with DVLM v6 - (Task #61)

- also removed from concept v1.0

---
Task #61: Regenerate Concepts due to change in DVLM

SaMuellerDLR commented:
@PhilMFischer does it make sense to have a CEF-X v1.1 even if we have never released a CEF-X v1.0?

PhilMFischer commented:
> @PhilMFischer does it make sense to have a CEF-X v1.1 even if we have never released a CEF-X v1.0?

Not sure? What is your opinion?

Commit:
Revert "Migrated CEFX concept to DVLM v7 - (Task #61)"

This reverts commit b7ab3c887724bdf3410b2a374d7fbb7be9760ce5.

Commit:
Regenerated CEFX Concept - (Task #61)

- Regenerated CEF-X Concept on version 1.0 since no version 1.0 was ever
released before.

---
Task #61: Regenerate Concepts due to change in DVLM

Commit:
Regenerated CEF Concept - (Task #61)

- Regenerated as Version 1.6

---
Task #61: Regenerate Concepts due to change in DVLM

Commit:
Regenerated CEF Interfaces concept - (Task #61)

- Regenerated as version 1.2 on DVLM v7

---
Task #61: Regenerate Concepts due to change in DVLM

PhilMFischer commented:
Migartion of Concepts is not needed. Thus it has to be redone. Only an update of DVLM is required but no update of the concept.
